src/lib.rs

```diff
@@ -222,6 +228,11 @@ fn initial_loading(_ctx: &InlineCtx) {
     check_input_on_boot();
 
     // let arc = resource::arc();
+    unsafe {
+        mount_cache_storage(skyline::c_str("cache\0"))
+    }
```

Comment:
Address the previous comment and call your Rust wrapper from nnsdk-rs here instead (that way we don't deal unsafe code directly)